fmt-tests/basics/buildfile

```diff
@@ -0,0 +1,45 @@
+include ../test-main/
+
+#test_names =
+#  args-test \
+#  assert-test \
+#  base-test
+gtest_test_names = args-test assert-test base-test chrono-test color-test compile-fp-test compile-test format-test gtest-extra-test noexception-test os-test ostream-test printf-test std-test unicode-test xchar-test
```

Comment:
Could this list have been formed from globing the `.cc` files? If not consider adding a comment about where to look to update this list (in `upstream/tests/CMakeFiles.txt`)